qemu/tests/qemu_img_measure_convert_image.py

```diff
@@ -34,8 +35,14 @@ def _get_img_obj_and_params(tag):
 
     def _qemu_io(img, cmd):
         """Run qemu-io cmd to a given img."""
-        logging.info("Run qemu-io %s" % img.image_filename)
-        q = QemuIOSystem(test, params, img.image_filename)
+        image_filename = img.image_filename
+        logging.info("Run qemu-io %s" % image_filename)
+        if img.image_format == "luks":
+            image_secret_object = img._secret_objects[0]
```

Comment:
if qemu-io doesn't need the image's backings, we can use img._secret_objects[-1] here, or we have to define all secret objects

Reply:
img._secret_objects[-1]  is ok, I will update it. thanks